tests/end_to_end_test/snapshot_create_test.go

```diff
@@ -74,6 +74,18 @@ func TestSnapshotCreate(t *testing.T) {
 
 	sources := clitestutil.ListSnapshotsAndExpectSuccess(t, e)
 	require.Len(t, sources, 3)
+
+	// test ignore-identical-snapshot
+	e.RunAndExpectSuccess(t, "policy", "set", "--global", "--ignore-identical-snapshot", "true")
+	_, outErr := e.RunAndExpectSuccessWithErrOut(t, "snapshot", "create", sharedTestDataDir2)
```

Comment:
i would expect this to succeed and compare list of snapshots before and after.

Reply:
compare the list of blobs as well (`kopia blob ls` before and after), since we're building this feature to avoid creating new blobs when snapshots are identical.